### PR TITLE
Added ingress for mojaloop

### PR DESCRIPTION
**Pull request by danishjamal104**
## Description

* Added ingress template for `ph-ee-connector-mojaloop-java`.

[PHEE-183](https://fynarfin.atlassian.net/browse/PHEE-183?atlOrigin=eyJpIjoiNWM4ZmYyZGVkZTlmNDgzNWExYmQ1ZDY3NTI3YTUwMjciLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[PHEE-183]: https://fynarfin.atlassian.net/browse/PHEE-183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helm/ph-ee-engine/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.4.0"
 description: PaymentHub EE Engine
 name: ph-ee-engine
-version: 1.4.0-SNAPSHOT
+version: 0.0.0-SNAPSHOT
```
Comment (fynmanoj) [inline helm/ph-ee-engine/Chart.yaml:5]: donot push this change

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/ph-ee-engine/Chart.yaml (hunk 1 above)
Comment (danishjamal104) [reply helm/ph-ee-engine/Chart.yaml:5]: I did this change for testing, if you remeber from the discussion with @avikganguly01  regarding testing the changes in ph-ee-env-template. Like cuse the version 0.0.0 and override the same image for all the testing.

Comment (fynmanoj) [reply helm/ph-ee-engine/Chart.yaml:5]: You can do this for testing. but do not send it upstream

Comment (danishjamal104) [reply helm/ph-ee-engine/Chart.yaml:5]: ok

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**